cleanup/macparakeet_cleanup/cli.py

```
@@ -0,0 +1,230 @@
+        alive, spawned = ensure_daemon(args.socket, model=args.model, debug=args.debug)
+        if not alive:
+            if args.debug:
+                sys.stderr.write("[cleanup] daemon auto-spawn failed\n")
+            return False
+        if spawned:
+            if args.debug:
+                sys.stderr.write("[cleanup] daemon spawned; warming up\n")
+            try:
+                send_warmup(args.socket, timeout=2.0)
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)